lunabot_embedded/firmware/teensy_main/lib/lunabot_drivers/RobotMsgs.pb.h

```diff
@@ -20,16 +20,14 @@ typedef struct _RobotSensors {
   float drive_left_ang;
   float drive_right_ang;
   float dep_ang;
-  float lead_screw_ang;
-  float act_ang;
-  float dep_weight;
   float uwb_dist_0;
   float uwb_dist_1;
   float uwb_dist_2;
+  float load_cell_0;
+  float load_cell_1;
 } RobotSensors;
 
 typedef struct _RobotEffort {
-  int32_t lead_screw;
   int32_t lin_act;
   int32_t left_drive;
   int32_t right_drive;
```

Comment:
Reasoning for making these deletions in this PR? (FYI: it breaks robot.cpp and teensy_main.ino)
it would be good to add a comment referencing a spreadsheet from P&H with the respective sensors/effort. can be hard to verify without and would be useful for others sanity checking as assembly is being done. I previously did this [here](https://github.com/PurdueLunabotics/purdue_lunabotics/blob/83bf7e900872154119562776a5196088808e6d79/lunabot_embedded/firmware/teensy_main/lib/lunabot_drivers/robot.cpp#L3)

Reply:
argh. thought I fixed those breaking bugs.
These changes were originally made in a separate branch, but I couldn't finish the load cell code without them. Do you know if a similar spreadsheet exists this year? Or if I should make one from the schematic myself?

Reply:
Should be good now. Waiting for build to complete

Reply:
Fixed. Don't think there are any other breaking changes in robot.cpp and teensy_main? I think I updated those.